### DISCUSSION TITLE
Tell me your favorite sports

### DISCUSSION THREAD

**Discussion by ibu00024** [NAIST-SE/AttractingContributors#1122] [2024 Attraction Contributions]
https://github.com/ibu00024/AttractingContributors

Please make contribute and tell me your favorite sports.

**Comment by mustafizur-r** [comment]
Hello [@ibu00024 ](https://github.com/ibu00024) , I recently made a contribution to your project , I have added “Cricket" .
could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.
